lib/widget/chatbot/chat_training.dart

```diff
@@ -138,6 +411,16 @@ class _ChatTrainingState extends State<ChatTraining> {
             return _digestiveAdvice();
           } else if (widget.answer == "การฟื้นฟูสมรรถภาพปอด") {
             return _pulmonaryAdvice();
+          } else if (widget.answer == "การดูแลสายระบาย") {
+            return _drainSecretionAdvice();
+          } else if (widget.answer == "การปฎิบัติตัวที่เหมาะสมก่อนกลับบ้าน") {
+            return _behaveAdvice();
+          } else if (widget.answer == "การส่งเสริมการหายของแผล") {
+            return _infectionAdvice();
+          } else if (widget.answer == "การลุกออกจากเตียง") {
+            return _nutritionAdvice();
+          } else if (widget.answer == "การปฎิบัติกิจวัตรประจำวัน") {
+            return _dailyActivityAdvice();
```

Comment:
consider using Switch in this case

Reply:
Can't change it because it seems to have onPressed in order to calling void function work but this answer we got from dialogflow.
The result below
![Screen Shot 2564-03-04 at 10 14 03](https://user-images.githubusercontent.com/57941667/109906049-5a9ac180-7cd2-11eb-96da-f9656d7d88bb.png)